src/ansys/geometry/core/tools/prepare_tools.py

```diff
@@ -168,6 +169,37 @@ def extract_volume_from_edge_loops(
             self._grpc_client.log.info("Failed to extract volume from edge loops...")
             return []
 
+    @protect_grpc
+    @min_backend_version(25, 2, 0)
+    def remove_rounds(self, faces: list["Face"]):
```

Comment:
Whether we return the response status (boolean) or nothing (None) let's add a typehint. And adapt the docstring if we return the boolean.